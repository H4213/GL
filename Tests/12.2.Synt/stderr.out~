Erreur récupérée (virugle manquant)
Erreur récupérée (egal manquant)
