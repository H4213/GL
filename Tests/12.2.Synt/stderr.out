Erreur récupérée (virgule manquant)
Erreur récupérée (egal manquant)
